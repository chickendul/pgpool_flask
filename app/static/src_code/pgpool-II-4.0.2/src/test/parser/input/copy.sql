version https://git-lfs.github.com/spec/v1
oid sha256:ed5309a8f96e9298fb5fd34618e259c3667920c3becca7722e8db87b9eb3372b
size 523
