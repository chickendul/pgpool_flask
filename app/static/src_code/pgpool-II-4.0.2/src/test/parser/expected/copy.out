version https://git-lfs.github.com/spec/v1
oid sha256:8185f007e824cbf192502a3d9af348ebbc4f23b65cd7520f89d35dfb573a1c48
size 566
